extensions/arc/runtime/src/main/java/io/quarkus/arc/properties/IfBuildProperty.java

```diff
@@ -30,4 +35,12 @@
      * Determines if the bean is to be enabled when the property name specified by {@code name} has not been specified at all
      */
     boolean enableIfMissing() default false;
+
+    @Retention(RetentionPolicy.RUNTIME)
+    @Target({ ElementType.METHOD, ElementType.TYPE, ElementType.FIELD })
+    @interface Container {
```

Comment:
Name it `List` to be similar to the pattern used in the Validator Framework?

Reply:
Which framework? `javax.validation`? Hm, I find the `Container` a better name for a containing annotation. Also I don't think it really matters because this annotation should not be used directly. That said, I'm ok with `IfBuildProperty.List`... so let's rename it ;-)